app/views/events/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Event information</h1>
+<p>Event Creator: <%= @event.user.name %></p>
+<h2>Attendees</h2>
+<ul>
+  <% @event.attendances.each do |a| %>
+    <li><%= User.find_by(id: a.user_id).name %></li>
```

Comment:
- [ ] Same here for business logic `User.find_by` shouldn't be in views.
Also we can simpley loop through `@event.attendees`, you have already defined an association for it in `event.rb`:
`has_many :attendees, through: :attendances`